lib/yeoman/templates/lib/capistrano/deploy/local.rb

```diff
@@ -3,4 +3,4 @@
   user:         fetch(:user),
   ssh_options:  fetch(:ssh_options)
 
-set :wp_path,   "/vagrant/web/wp"
```

Comment:
While this _does_ modify a generated file, it has no functional difference and should not constitute a bc break